doc/recipe/preprocessor.rst

```diff
@@ -867,6 +877,67 @@ See also :func:`esmvalcore.preprocessor.regrid`
    for resolutions of ``< 0.5`` degrees the regridding becomes very slow and
    will use a lot of memory.
 
+.. _generic regridding schemes:
+
+Generic regridding schemes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`Iris' regridding <iris:interpolation_and_regridding>` is based around the
+flexible use of so-called regridding schemes. These are classes that know how
+to transform a source cube with a given grid into the grid defined by a given
+target cube. Iris itself provides a number of useful schemes, but they are
+largely limited to work with simple, regular grids. Other schemes can be
+provided independently. This is interesting when special regridding-needs arise
+or when more involved grids and meshes need to be considered. Furthermore, it
+may be desirable to have finer control over the parameters of the scheme than
+is afforded by the built-in schemes described above.
+
+To facilitate this, the :func:`~esmvalcore.preprocessor.regrid` preprocessor
+allows the use of any scheme designed for Iris. The scheme must be installed
+and importable. To use this feature, the ``scheme`` key passed to the
+preprocessor must be a dictionary instead of a simple string that contains all
+necessary information. That includes a ``reference`` to the desired scheme
+itself, as well as any arguments that should be passed through to the
+scheme. For example, the following shows the use of the built-in scheme
+:class:`iris.analysis.AreaWeighted` with a custom threshold for missing data
+tolerance.
+
+.. code-block:: yaml
+
+    preprocessors:
+      regrid_preprocessor:
+        regrid:
+          target_grid: 2.5x2.5
+          scheme:
+            reference: iris.analysis:AreaWeighted
+            mdtol: 0.7
+
+The value of the ``reference`` key has two parts that are separated by a
+``:`` with no surrounding spaces. The first part is an importable Python
+module, the second refers to the scheme, i.e. some callable that will be called
+with the remaining entries of the ``scheme`` dictionary passed as keyword
+arguments.
```

Comment:
```suggestion
argument.
```

Reply:
well, it's one dictionary = one keyword argument, but am not fussed, keep it as it is, me and grammatical correctness are not friends anyway

Reply:
ah gotcha, cheers, man! All clear for take off :airplane: